tools/sphere.c

```diff
@@ -69,6 +69,17 @@ int do_sloop( string arg ) {
                 ,FINDP( arg )->query_name() ), BOLD_PURPLE );
         return 1;
     }
+    /**
+     * If already slooping victim, instead remove both shadows
+     */
+    if ( sizeof( find_slooper_shadow( slooper ) ) && sizeof( find_sloopee_shadow( sloopee ) ) ) {
```

Comment:
It also seems like a person can get multiple sloopee shadows if slooped by multiple people. Was that your intention?